rel/i18n/emqx_schema.hocon

```diff
@@ -412,25 +412,29 @@ fields_ws_opts_mqtt_piggyback.desc:
 fields_ws_opts_mqtt_piggyback.label:
 """MQTT Piggyback"""
 
-base_listener_mountpoint.desc:
-"""When publishing or subscribing, prefix all topics with a mountpoint string.
-The prefixed string will be removed from the topic name when the message
-is delivered to the subscriber. The mountpoint is a way that users can use
-to implement isolation of message routing between different listeners.
-For example if a client A subscribes to `t` with `listeners.tcp.\<name>.mountpoint`
-set to `some_tenant`, then the client actually subscribes to the topic
-`some_tenant/t`. Similarly, if another client B (connected to the same listener
-as the client A) sends a message to topic `t`, the message is routed
-to all the clients subscribed `some_tenant/t`, so client A will receive the
-message, with topic name `t`.<br/>
-Set to `""` to disable the feature.<br/>
-
-Variables in mountpoint string:
-  - <code>${clientid}</code>: clientid
-  - <code>${username}</code>: username"""
-
-base_listener_mountpoint.label:
-"""mountpoint"""
+base_listener_mountpoint.desc: """~
+    Automatically prefixes topics for clients connected to this listener.
+    - Adds a static or templated prefix (e.g., `n1/` or `${username}/`) to topics sent by clients.
+    - Removes the prefix from topics sent to clients.
```

Comment:
Maybe just this, instead of separate bullet point list at the bottom?
```suggestion
    - Adds a static or templated prefix (e.g., `n1/` or `${username}/`) to topics used by clients in `SUBSCRIBE` and `UNSUBSCRIBE` requests, `PUBLISH` messages, and Will Message (if supplied in the `CONNECT` packet).
    - Removes this prefix from topics of messages published to the respective subscriptions.
```